lib/containers/artist.js

```diff
@@ -54,6 +62,12 @@ export default Relay.createContainer(Artist, {
   fragments: {
     artist: () => Relay.QL`
       fragment on Artist {
+        partner_shows {
+          _id
+        }
```

Comment:
I worry that this might lead to much more data being fetched, which is why I suggested to use those query fragments that we already use: https://github.com/artsy/emission/pull/76#issuecomment-221089319

However, the best solution imo would be to make metaphysics be able to return a shows count, like you did for the artworks. Have you checked if it doesn’t provide that yet?